src/mrpro/data/AcqInfo.py

```diff
@@ -258,3 +259,51 @@ def spatialdimension_2d(data: np.ndarray) -> SpatialDimension[torch.Tensor]:
             version=tensor_2d(headers['version']),
         )
         return acq_info
+
+    def add_to_ismrmrd_acquisition(
+        self, acquisition: ismrmrd.Acquisition, other: type_utils.TorchIndexerType, k2: int, k1: int
```

Comment:
I guess this should be 
`other: tuple[_SingleTorchIndexerType, ...]`

would it make sense to make `_SingleTorchIndexerType` from utils.typing also globally available?